maven-central-sync/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+
+LOCAL_REPO_DIR=$1
+
+CMD="mvn org.sonatype.plugins:nexus-staging-maven-plugin:1.6.7:"
+DEFAULT_OPTS="-DnexusUrl=https://s01.oss.sonatype.org/ -DserverId=ossrh"
+PROFILE_OPT="-DstagingProfileId=13c1877339a4cf"
+
+open="$CMD:rc-open $DEFAULT_OPTS $PROFILE_OPT"
+deploy="$CMD:deploy-staged-repository $DEFAULT_OPTS $PROFILE_OPT"
+close="$CMD:rc-close $DEFAULT_OPTS"
+release="$CMD:rc-release $DEFAULT_OPTS"
+
+STAGING_REPO=$($open -DopenedRepositoryMessageFormat="R3P0:%s" | grep "R3P0:" | cut -d: -f2)
```

Comment:
Looks like a magic command, what does this R3P0 do?

Reply:
magic 🧙🏻 

With the property `openedRepositoryMessageFormat` you can specify a custom pattern for printing the staging repo ID. This prefix `R3P0` is used in the grep command to get the correct line.